pennylane/fermi/fermionic.py

```diff
@@ -159,6 +159,101 @@ def __pow__(self, value):
         return operator
 
 
+# pylint: disable=too-few-public-methods
+class FermiC(FermiWord):
+    r"""FermiC(wire)
+    The fermionic creation operator :math:`\hat{c}^\dagger`
+
+    .. math:: \hat{c}^\dagger \ket{n} = \sqrt{n + 1} \ket{n+1}
+
+    **Details:**
+
+    * Number of wires: 1
+    * Number of parameters: 0
+
+    Args:
+        wire(int): the positive integer indicating the energy level the operator acts on. The lowest
+            energy level is designated 0, and subsequent energy levels are numbered chronologically.
```

Comment:
Is this accurate? And is it too specific to talk about 'energy levels'?

Reply:
We can replace `energy level` with `orbital` or just `state` to be more general. For instance, `a^{\dagger}_3` operates on the 3rd orbital or 3rd state in `|0000>`.